content/docs/for-buildpack-authors/how-to/write-buildpacks/specify-env.md

```diff
@@ -3,15 +3,108 @@ title="Specify the environment"
 weight=99
 +++
 
+Environment variables are a common way to configure various buildpacks at build-time.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+### POSIX Path Variables
+
+The following layer path environment variables MUST be set by the `lifecycle` during the `build` and `launch` phases in order to make buildpack dependencies accessible.
+
+During the `build` phase, each variable designated for build MUST contain absolute paths of all previous buildpacks’ `<layers>/<layer>/` directories that are designated for build.
+
+When the exported OCI image is launched, each variable designated for launch MUST contain absolute paths of all buildpacks’ `<layers>/<layer>/` directories that are designated for launch.
+
+In either case,
+
+- The lifecycle MUST order all `<layer>` paths to reflect the reversed order of the buildpack group.
+- The lifecycle MUST order all `<layer>` paths provided by a given buildpack alphabetically ascending.
```

Comment:
Let's make a section for "when multiple buildpacks define the same variable" or something like that. That would be a good introduction to the modification rules.

Reply:
added